storage-api/src/java/org/apache/hadoop/hive/common/ValidTxnList.java

```diff
@@ -31,6 +31,11 @@
    */
   public static final String VALID_TXNS_KEY = "hive.txn.valid.txns";
 
+  /**
+   * Key used to store txn id for compactor in a
+   * {@link org.apache.hadoop.conf.Configuration} object.
+   */
+  public static final String COMPACTOR_VALID_TXNS_ID_KEY = "hive.compactor.txn.valid.txns.id";
```

Comment:
I don't see the place in the code which sets this key in the configuration object.

Reply:
Fixed it